### DISCUSSION TITLE
getting company info and api token for http requests

### DISCUSSION THREAD

**Discussion by durandal2** [fattureincloud/api#374] [Q&A] [answered]
Hallo
look at his example found in the support documentation:

![image](https://github.com/fattureincloud/api/assets/17932395/6d2be473-c171-4305-a362-94f80a6a3791)

I cannot find information on how to get the company id number and api access token required.
can you help?

thank you

**Comment by valmoz** [comment, marked as answer]
Hello,
I suggest you check our [Step-by-step guide](https://developers.fattureincloud.it/docs/step-by-step) or our [FAQs page](https://developers.fattureincloud.it/docs/FAQs) to find this kind of information.

Also, you can try to check the search bar on top of our pages, and also other discussions on this page (remember to check also the closed discussions).

Thanks
Mauro
